test/e2e/mpi_job_test.go

```diff
@@ -211,12 +231,31 @@ var _ = ginkgo.Describe("MPIJob", func() {
 	})
 })
 
+func resumeJob(mpiJob *kubeflow.MPIJob) *kubeflow.MPIJob {
```

Comment:
For follow up: accept contexts in all these functions

Reply:
Will open a PR directly after this, doesn't seem it requires Issue for later.

Reply:
Similarly, going to open a PR to copy the MPIJob conditions from common.

Reply:
@mimowo I opened the PR to copy the MPIJob conditions to this repo in #514 since I faced issues caused by the MPIJob conditions in #510.